configure.ac

```diff
@@ -213,6 +213,35 @@ AC_ARG_ENABLE(
     [enable_ml_tests="yes"],
     [enable_ml_tests="no"]
 )
+AC_ARG_ENABLE(
```

Comment:
Considering that we are failing to compile on `Slackware current`, `Alpine`, and now on 
[Raspberry OS](https://github.com/netdata/netdata/files/9874127/err.txt). I strongly suggest you to review this whole file. I am not going to review it right now.

Reply:
I will need some help with packaging as I said in the PR description but it shouldn't fail when you are trying to build on a Raspbian. Did you try to build it using the included [`run_stress_test.sh`](https://github.com/netdata/netdata/pull/13291/files#diff-6dee16a3677d914fe839626f9b35ace8f99b7177b6b58395424d6669a794bb67) script as `./run_stress_test.sh 1 0`? 

If you don't pass on the correct `CFLAGS` it won't be able to find some headers (as I imagine is the case for your Raspberry Pi).

Reply:
Of course, we can help you (ping @Ferroin, @underhood , and @vlvkobal  ). :) 
We need to adjust this before to merge, else we would need another way to compile netdata. And this would be an issue to make our packages and for users.

Reply:
This has been updated now to allow for correct configuration when `--enable-logsmanagement` is passed on to the installer.